blackbox/docs/administration/auth_methods.txt

```diff
@@ -0,0 +1,18 @@
+
+.. _auth_methods:
+
+======================
+Authentication Methods
+======================
+
+.. _auth_trust:
+
+Trust Method
+============
+
+When the ``trust`` method is used, CrateDB assumes that anyone is allowed access
+to the cluster. Therefore, this method allows any client to login as any existing
+CrateDB user (even the ``crate`` superuser) without specifying further credential
+information like passwords etc.
+
+The ``trust`` method can be used in conjunction with the :ref:`administration_hba`.
```

Comment:
in most cases: you might want to look at the last para in the trust docs https://www.postgresql.org/docs/current/static/auth-methods.html which states this as a reasoning